tools/install/src/getPackagesToInstall.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-console */
+import { ALL_PACKAGES } from './ALL_PACKAGES';
+import type { InstallCommandOptions } from './types';
+
+// Essential packages that provide core functionality
+const ESSENTIAL_PACKAGES = [
+  '@leafygreen-ui/leafygreen-provider',
+  '@leafygreen-ui/emotion',
+  '@leafygreen-ui/lib',
+];
+
+// Basic packages that include essentials plus commonly used components
+const BASIC_PACKAGES = [
+  ...ESSENTIAL_PACKAGES,
+  '@leafygreen-ui/banner',
+  '@leafygreen-ui/button',
+  '@leafygreen-ui/card',
+  '@leafygreen-ui/icon',
+  '@leafygreen-ui/icon-button',
+  '@leafygreen-ui/modal',
+  '@leafygreen-ui/tokens',
+  '@leafygreen-ui/typography',
+];
+
+/**
+ * Read package names from the static file
+ * Falls back to fetching from NPM if the file doesn't exist
+ */
+export function getPackagesToInstall(
+  explicitPackages: Array<string>,
+  options: InstallCommandOptions,
+): Array<string> {
+  const { verbose, essentials, basic, core, charts, chat } = options;
+
+  const availablePackages = ALL_PACKAGES;
+  verbose &&
+    console.log(`Found ${availablePackages.length} packages in static file`);
+
+  // If specific packages are requested, use those
+  if (explicitPackages.length > 0) {
+    return availablePackages
+      .filter(fullName => {
+        const pkgName = fullName.split('/')[1]; // Get name without scope
+        return explicitPackages.includes(pkgName);
+      })
+      .map(pkg => `${pkg}@latest`);
+  }
+
+  // Handle flag options to filter packages
+  let packagesToInstall: Array<string> = [];
+
+  // Filter by specific package sets based on flags
+  if (basic) {
```

Comment:
Should there be a warning that combining package and scope is not allowed? 

For example: `--basic --charts`